src/lib/setup.ts

```diff
@@ -156,9 +107,4 @@ export default function forceMiddleware(app) {
     // Mount desktop app
     app.use(require("../desktop"))
   }
-
-  // Ensure CurrentUser is set for Artsy Passport
-  // TODO: Investigate race condition b/t reaction's use of AP
-  // TODO: This is only run once, does it need to still happen
-  artsyPassport.options.CurrentUser = CurrentUser
```

Comment:
Does article authentication still work without this? That is the existing use case for reaction/passport.

Reply:
Yes